Move custom domain post job to Github Actions (#791)
kyma-project/api-gateway: docs/contributor/04-30-ci-cd.md

- 'restricted' - used when an outside collaborator runs a job in the repository. The run must be approved by
  @kyma-project/goat.

> ## Prerequisites Github Actions secrets

Reviewer: ```suggestion
## Prerequisites for GitHub Actions Secrets
```

> ### Gardener specific

Reviewer: ```suggestion
### Gardener Specific
```

> - `GARDENER_TOKEN` - Kubeconfig token used to provision test purpose shoots and later cleanup

Reviewer: ```suggestion
- `GARDENER_TOKEN` - Kubeconfig token used to provision test-purpose shoots and for later cleanup
```

> - `DNS_SECRET_JSON` - DNS ServiceAccount used to set up custom domain through Gardener capabilities (BASE64 Encoded)

Reviewer: ```suggestion
- `DNS_SECRET_JSON` - DNS ServiceAccount used to set up the custom domain using Gardener capabilities (Base64 encoded)
```
